config/webpack-prod.config.js

```diff
@@ -11,9 +11,7 @@ module.exports = merge(webpackCommon, {
     },
     output: {
         filename: '[name].js',
-        path: path.resolve(__dirname, '../dist'),
-        library: true,
```

Comment:
Any reason why we are removing this. Consumers will not be able to use this directly as a lib. Also need to check if the webpackbin demo continues to work with these changes